### PR TITLE
17 2 release notes

### PR DESCRIPTION
**Pull request by patflynn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+  - App Engine initialization and region selection. ([#1232](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1232))
+  
+### Fixed
+  - Stopping dev_appserver on Windows always fails with com.intellij.execution
+  .ExecutionException. ([#1215](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1215))
+  - New AE standard project wizard should generate web.xml with servlet 2.5. ([#1194](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1194))
+  - Clear datastore checkbox for app engine standard local server does not work. ([#1188](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1188))
+  - Don't show projects scheduled for deletion in the project selector. ([#1119](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1119))
+  
+Visit our [17.2 Release Milestone](https://github
```
Comment (etanshaul) [inline CHANGELOG.md]: link doesn't work. I think its a line break thing

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,28 @@
 All notable changes to this project will be documented in this file.
 ## [unreleased]
 
+## [17.2.1]
+
+Happy new year Cloud Tools for IntelliJ users! This year's first release is primarily a maintenance
+release.  If you are having authentication problems using Cloud Source Repositories and our
```
Comment (etanshaul) [inline CHANGELOG.md]: nit: two spaces after period before "If you are having .."

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -33,14 +33,43 @@
...
       This plugin requires the use of Cloud SDK v 133.0.0 for correct execution of the local
       development server with the latest Java 8 SDK. Please run 'gcloud components update' from your
       shell to ensure you have the latest Cloud SDK release.
       </p>
 
+      <h2>[17.2.1]</h2>
+
+      <p>Happy new year Cloud Tools for IntelliJ users! This year's first release is primarily a maintenance
+        release.  If you are having authentication problems using Cloud Source Repositories and our
+        plugin, check out <a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1174">this possible solution</a>.</p>
```
Comment (joaoandremartins) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: Solution is buried deep in the link. Can we link to something more precise?

Comment (etanshaul) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: you could link to the user's comment where he says he gets unblocked:

https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1174#issuecomment-275440330

File: CHANGELOG.md (hunk 2 above)
Comment (patflynn) [inline CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 1 above)
Comment (patflynn) [inline CHANGELOG.md]: fixed

**Comment by etanshaul** [comment]
what about calling this one out: https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/946

Since it may be weird to include this as a regular line item (since no change was needed), maybe we can call it out at the top instead?

I'm thinking this may be useful to see.

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml (hunk 3 above)
Comment (patflynn) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: the user should understand the problem and make sure they have the same one before they apply a solution, barring writing up a better problem statement and solution (which I'm not planning on doing), this is I think the best we can do.

Comment (etanshaul) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: that's true. sgtm

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,27 @@
 All notable changes to this project will be documented in this file.
 ## [unreleased]
 
+## [17.2.1]
+
+Happy new year Cloud Tools for IntelliJ users! This year's first release is primarily a maintenance
```
Comment (etanshaul) [inline CHANGELOG.md]: new year capitalized?

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,27 @@
...
+
+Here is a list of the visible changes:
+
+### Added
+  - Support for multiple cloud source repositories for a single GCP project. ([#1024](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1024))
+  - App Engine initialization and region selection. ([#1232](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1232))
+  
+### Fixed
+  - Stopping dev_appserver on Windows always fails with com.intellij.execution
+  .ExecutionException. ([#1215](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1215))
```
Comment (etanshaul) [inline CHANGELOG.md]: nit, extra space before ".ExecutionException"

**Pull request merged**